wagtail/core/fields.py

```diff
@@ -61,8 +96,10 @@ def deconstruct(self):
         return name, path, args, kwargs
 
     def to_python(self, value):
+        strategy = StreamFieldPrefetcher()
```

Comment:
Why name it `strategy` instead of `prefetcher`?

Reply:
The reason is explained at https://martinfowler.com/bliki/TwoHardThings.html ;-)